src/configs/cmd_duration.rs

```diff
@@ -0,0 +1,24 @@
+use crate::config::{ModuleConfig, RootModuleConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct CmdDurationConfig<'a> {
+    pub min_time: i64,
+    // TODO: Rewrite prefix with SegmentConfig
```

Comment:
Are we going to address this `TODO` before landing the PR?

Reply:
Oh sorry, I should probably open another PR instead of leaving a TODO comment here as it seems to be a new feature. Thanks for the question!